secret-store/Cargo.toml

```diff
@@ -17,6 +17,7 @@ ethcore-accounts = { path = "../accounts", optional = true}
 ethcore-call-contract = { path = "../ethcore/call-contract" }
 ethcore-sync = { path = "../ethcore/sync" }
 ethereum-types = "0.8.0"
+ethkey = { path = "../accounts/ethkey", optional = true }
```

Comment:
I had to add this too, to get `parity-ethereum` to build. A bit concerning CI didn't catch this.